addons/osio-addon/src/main/java/io/fabric8/launcher/osio/projectiles/context/OsioProjectileContext.java

```diff
@@ -37,6 +38,9 @@
     @DefaultValue("1.0.0")
     private String projectVersion;
 
+    @FormParam("dependency")
+    private List<String> dependencies;
```

Comment:
I think it would be better if you changed this to `List<Dependency> dependencies` and used a `ParamConverter` to convert the String to Dependency. Here is an example: https://github.com/fabric8-launcher/launcher-backend/commit/aedbebb8da169abb99e7bb38ccaf4bc3bb9a7cce

Reply:
Here is an explanation on how ParamConverters work: https://abhirockzz.wordpress.com/2016/01/20/using-parameter-converters-in-jax-rs/